src/plugins/template-renderer/filters/to-error-list.js

```diff
@@ -3,8 +3,17 @@ export const toErrorList = (errors) => {
     return []
   }
 
-  return Object.entries(errors).map(([field, error]) => ({
-    text: error.text,
-    href: `#${field}`
-  }))
+  const uniqueFields = new Set()
+
+   return Object.entries(errors)
+    .reduce((acc, [field, error]) => {
+      if (!uniqueFields.has(error.text)) {
+        uniqueFields.add(error.text)
+        acc.push({
+          text: error.text,
+          href: `#${field}`
+        })
+      }
+      return acc
+    }, [])
```

Comment:
This depends on what you want to happen but IF you don't want the error message to appear multiple times at the summary at the top, then this will remove all duplicates.

This does mean that if only one error is included, then it will only link to the first missing property.

Not sure what your intended behaviour is.